source/globalCommands.py

```diff
@@ -2177,6 +2177,24 @@ def script_reportStatusLine(self, gesture):
 		else:
 			self.script_copyStatusLine(gesture)
 
+	@script(
+		description=_(
+			# Translators: Description for a keyboard command which reports the
+			# accelerator key of the currently focused object.
+			"Reports the shortcut key of the currently focused object."
+		),
+		category=SCRCAT_FOCUS,
+	)
+	def script_reportFocusObjectAccelerator(self, gesture: "inputCore.InputGesture") -> None:

```

Comment:
my mistake - this does not need to be quoted

```suggestion
	def script_reportFocusObjectAccelerator(self, gesture: inputCore.InputGesture) -> None:
```